keyboards/rgbkb/common/touch_encoder.c

```diff
@@ -244,6 +244,14 @@ void touch_encoder_update_slave(slave_touch_status_t slave_state) {
 }
 
 void touch_encoder_update(int8_t transaction_id) {
+    if (is_keyboard_master()) {
```

Comment:
Does it matter that this is now not throttled by `TOUCH_UPDATE_INTERVAL`?

Reply:
doh, good catch, will fix that.

Reply:
fixed in [f62bda8](https://github.com/qmk/qmk_firmware/pull/16952/commits/f62bda807817c481aa81a2320d4ef31aac641dd0)